### DISCUSSION TITLE
3 pin headers - what are they for please

### DISCUSSION THREAD

**Discussion by n9jcv** [picocomputer/community#67] [Q&A] [answered]
In the soldering build, there are two 3 pin headers under the PI.  I am struggling to understand what they are for.  I do not see how anything can be connected to them.  I must have missed something??

**Comment by WojciechGw** [comment, marked as answer]
SWD debug interface, middle one for Raspberry Pico W (with WiFi) and the other one for regular Raspberry Pico
look at the schema (U2 Pico RIA)
connected by PCB to Pico VGA : 
pin 34 (GP28) SWDIO
pin 29 (GP22) SWCLK

**Reply by n9jcv** [reply]
Thanks for the answer and your time!

**Reply by vruumllc** [reply]
I think these are obsolete now that the vga pico is no longer used for debugging.

**Reply by WojciechGw** [reply]
@n9jcv yw :) anytime

@vruumllc
indeed
personaly, I removed both connectors and solder 3-pin headers to both picos for debug probes
